evennia/commands/default/unloggedin.py

```diff
@@ -118,6 +117,45 @@ def guest_player(session):
         return True, new_player
 
 
+def normal_player(session, name, password):
```

Comment:
A more instructive name would be `create_normal_player`. This also needs docstring.